### PR TITLE
Feature/client post

### PR DESCRIPTION
**Pull request by Wils0nC00k**

### CODE REVIEW

File: routes/clients.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const router = express.Router();
 const clients = require('../data/clients.json');
+const formidable = require('formidable');
```
Comment (laura-mso) [inline routes/clients.js]: As far as I see we are not using formidable below?

Comment (Wils0nC00k) [reply routes/clients.js]: Oh yeah because we don't have database or a cloud, one sec

File: routes/clients.js [hunk 2]
```diff
@@ -18,7 +18,13 @@ router.get('/:id', function(req, res, next) {
 });
 
 router.post('/', (req, res, next) => {
-  const clientData = req.body;
+  const newClientId = clients[clients.length - 1].id + 1;
+  const clientData = {
+    id: newClientId,
+    name: req.body.name,
+    initials: req.body.initials,
+    contact_information: req.body.contact_information,
```
Comment (laura-mso) [inline routes/clients.js]: I am wondering what kind of data contact_information is

Comment (Wils0nC00k) [reply routes/clients.js]: https://lesewert.herokuapp.com/api/v1/clients

Comment (Wils0nC00k) [reply routes/clients.js]: I have a nested state that captures the value with just this key

**Pull request merged**